### PR TITLE
Fixed example

### PR DESCRIPTION
**Pull request by sobolev-alexey**

### CODE REVIEW

**Comment by sobolev-alexey** [comment]
https://alnodes.devnet.iota.org:443 is a load balancer. Multiple messages sent one after another can't be fetched if redirected to another node.
```
run(message1)
  .then(() => {
    console.log("message 1 done")
    
    run(message2)
      .then(() => {
        console.log("message 2 done")

        run(message3)
          .then(() => console.log("message 3 done"))
          .catch((err) => console.error(err));
      })
      .catch((err) => console.error(err));
  })
  .catch((err) => console.error(err));
```

**Comment by sobolev-alexey** [comment]
Default fetch output as 
```
{ nextRoot: 'HASGKMTWUPQYMGCIMTJITUNMECTHCVIQZQYQSOJOPMUYIOHSYUUIHFVMJNBBFOQZGXMLFYJVLAAOGCWKN',
  message: 'ODGAADTCGDGDPCVCTCGADBGAWBTCGDGDPCVCTCEAUCFDCDADEALBCDQCGAQAGAHDXCADTCGDHDPCADDDGADBGAVAVATAWAVATAWAUAVACBQAEAWADBXACBDBWAWAEAZBWBGAQD',
  tag: 'MY9MAM999999999999999999999' }
``` 
seem to be less useful. I added a trytes to ascii converter on top to get plain text message

**Comment by obany** [comment]
I didn't want to have any JSON conversion in this simple example as that's a secondary encoding. trytes conversion addition is fine.

File: examples/simple/index.js [hunk 1]
```diff
@@ -1,53 +1,68 @@
...
 
     // Display the details for the MAM message.
     console.log('Address:', mamMessage.address);
     console.log('Root:', mamMessage.root);
     console.log('NextRoot:', mamMessage.nextRoot);
-    console.log('Payload:', mamMessage.payload);
+    console.log('Seed:', seed);
 
     // Decode the message using the root and sideKey.
-    // This is only for demonstration purposes, there is no reason to decode at this point
```
Comment (obany) [inline examples/simple/index.js]: Please leave this comment in

File: examples/simple/index.js [hunk 2]
```diff
@@ -57,9 +72,14 @@ async function run() {
     // Try fetching it as well.
     console.log('Fetching from tangle, please wait...');
     const fetched = await mamFetch(api, mamMessage.root, mode, sideKey)
-    console.log('Fetched', fetched);
+    fetched && console.log('Fetched', JSON.parse(trytesToAscii(fetched.message)));
```
Comment (obany) [inline examples/simple/index.js]: Remove any JSON encoding/decoding, want this to be a very simple example

File: examples/simple/index.js [hunk 3]
```diff
@@ -1,53 +1,68 @@
...
+};
+
+async function run(message) {
     // Setup the details for the channel.
-    const seed = 'ENTER YOUR SEED HERE';
     const mode = 'restricted';
     const sideKey = 'MYKEY';
-    const message = 'MY9MESSAGE';
+    const seed = generateSeed();
+    const trytes = asciiToTrytes(JSON.stringify(message))
```
Comment (obany) [inline examples/simple/index.js]: Remove any JSON encoding/decoding, want this to be a very simple example

**Comment by sobolev-alexey** [comment]
> I didn't want to have any JSON conversion in this simple example as that's a secondary encoding. trytes conversion addition is fine. Don't want devs to think you must have JSON

Don't you think that storing JSON payload is a way more common use case for MAM streams? 
[almost] every PoC we developed stored complex objects, not plain strings. 
To make the example more close to the real use case, I'd leave a simple JSON structure as payload.

**Comment by obany** [comment]
You could add a second example with JSON bits, but I wanted the simple example to almost be the same as the api reference example in the readme.

**Pull request merged**